py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2648,10 +2648,15 @@ def topk_validator(node: Node) -> bool:
 
 
 def sort_validator(node: Node) -> bool:
-    shape = node.args[0].meta.get("tensor_meta").shape
+    meta_data = node.args[0].meta.get("tensor_meta")
+    if meta_data is None:
+        return False
+    shape = meta_data.shape
     dim = node.args[1]
     dim = get_positive_dim(dim, len(shape))
     k = shape[dim]
+    if not isinstance(k, int):
```

Comment:
If dim k is static and other dims are dynamic, we can support sort ops.
This is to validate if k is static or not.

Reply:
Thanks for the comment. Is it possible that k is -1?

Reply:
Thanks for bringing this to my attention.
I checked only dynamic dims are passed to export() in torch_tensorrt.dynamo.trace()
https://github.com/pytorch/TensorRT/blob/main/py/torch_tensorrt/dynamo/_tracer.py#L81
So I think static dim value in meta_data will be valid dim value(>0). If you think static shape dim value from meta data could be -1, I will update to validate this case as well.

+elif isinstance(k, int) and k < 0:
+    return False